chrome/content/zotero/searchDialog.js

```diff
@@ -40,7 +40,9 @@ function doLoad()
 	var searchBox = document.getElementById('search-box');
 	searchBox.groups = io.dataIn.groups;
 	searchBox.search = io.dataIn.search;
-	document.getElementById('search-name').value = io.dataIn.name;
+	let searchName = document.getElementById('search-name');
+	searchName.value = io.dataIn.name;
+	searchName.setSelectionRange(0, searchName.value.length, "backward");
```

Comment:
`searchName.select()`?

Reply:
Oh cool, I didn't realize that `.select` works just as well as `.setSelectionRange`. Good to know!